eu.octanne.McBoyard.yml

```
@@ -0,0 +1,64 @@
+app-id: eu.octanne.McBoyard
+separate-locales: false
+
+base: io.atom.electron.BaseApp
+base-version: '19.08'
+
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+
+  - --filesystem=home
+
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+  - --env=JAVA=/app/jre/bin/java
+
+build-options:
+  env:
+    JAVA_HOME: /usr/lib/sdk/openjdk11/
+
+modules:
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://www.x.org/archive/individual/app/xrandr-1.5.1.tar.gz
+        sha256: 7b99edb7970a1365eaf5bcaf552144e4dfc3ccf510c4abc08569849929fb366e
+    cleanup:
+      - /share/man
+
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: mcboyard
+    buildsystem: simple
+    build-commands:
+      - install -Dp -m755 McBoyard.jar /app/bin/McBoyard.jar
+      - install -Dp -m755 mcboyard.sh /app/bin/mcboyard.sh
+      - install -Dp -m644 eu.octanne.McBoyard.png /app/share/icons/hicolor/256x256/apps/eu.octanne.McBoyard.png
+      - install -Dp -m644 eu.octanne.McBoyard.desktop /app/share/applications/eu.octanne.McBoyard.desktop
+      - install -Dp -m644 eu.octanne.McBoyard.appdata.xml /app/share/appdata/eu.octanne.McBoyard.appdata.xml
```

```suggestion
      - install -Dp -m644 eu.octanne.McBoyard.metainfo.xml /app/share/metainfo/eu.octanne.McBoyard.metainfo.xml
```
you should use metainfo instead of appdata, it's the recommended directory per appstream specs